Fix -Wimplicit-function-declaration error with gcc 14.

Error:

```
src/tilda-lock-files.c: In function 'getPids':
src/tilda-lock-files.c:223:22: error: implicit declaration of function 'popen';
did you mean 'open'? [-Wimplicit-function-declaration]
  223 |     if ((ps_output = popen (ps_command, "r")) == NULL) {
      |                      ^~~~~
      |                      open
```

--- tilda-tilda-2.0.0-origin/src/tilda-lock-files.c
+++ tilda-tilda-2.0.0/src/tilda-lock-files.c
@@ -13,6 +13,7 @@
  * License along with this library. If not, see <http://www.gnu.org/licenses/>.
  */
 
+#define _DEFAULT_SOURCE
 #include "tilda-lock-files.h"
 
 #include "debug.h"
